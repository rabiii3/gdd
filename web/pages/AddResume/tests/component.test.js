import React from 'react';
import { shallow } from 'enzyme';
import { Page } from '..';
import { IntlProvider } from 'react-intl';

const intlProvider = new IntlProvider({ locale: 'en', messages: {} }, {});
const { intl } = intlProvider.getChildContext();

describe('web | components | AddResume | component', () => {
  describe('render', () => {
    it('should match snapshot', () => {
      const props = {
        classes: {},
        actions: { goHome: new Function() },
        location: { state: { file: {} } },
        intl: {
          formatMessage: jest.fn()
        },
      };
      const wrapper = shallow(<Page {...props} intl={intl} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
