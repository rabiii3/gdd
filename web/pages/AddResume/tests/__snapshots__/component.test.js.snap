// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`web | components | AddResume | component render should match snapshot 1`] = `
<WithStyles(Container)>
  <WithStyles(StyledHeader)>
    <WithStyles(StyledHeaderLeft)>
      <withRouter(ArrowBack) />
      <Spacer
        size="SMALL"
      />
      <WithStyles(Title)
        text="Add Resume"
      />
    </WithStyles(StyledHeaderLeft)>
    <WithStyles(StyledHeaderRight) />
  </WithStyles(StyledHeader)>
  <WithStyles(Paper)
    elevation={2}
  >
    <WithStyles(Connect(withState(withHandlers(withStateHandlers(InjectIntl(withRouter(EnhancedComponent)))))))
      actions={
        Object {
          "goHome": [Function],
        }
      }
      isEditable={true}
      resume={
        Object {
          "file": undefined,
        }
      }
    />
  </WithStyles(Paper)>
</WithStyles(Container)>
`;
